### PR TITLE
jenkins: Visual updates

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

* Replaces GitHub logo with Jenkins logo (hat tip @regicsolutions who noted this in #4328, and ref https://www.jenkins.io/press/#branding )
* Moves to use new Backstage `<Breadcrumb>` component over Material-UI's
* Prefer term "Project" ver "Job" per deprecation notice in [Jenkins Glossary](https://www.jenkins.io/doc/book/glossary/)

| Before | After |
| --- | --- |
| image |  image |
|  image |  image |

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

File: plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx [hunk 1]
```diff
@@ -231,9 +231,9 @@ export const CITableView = ({
       onChangeRowsPerPage={onChangePageSize}
       title={
         <Box display="flex" alignItems="center">
-          <GitHubIcon />
-          <Box mr={1} />
-          <Typography variant="h6">{projectName}</Typography>
+          <img src={JenkinsLogo} alt="Jenkins logo" height="50px" />
```
Comment (adamdmharvey) [inline plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx:234]: I know this is a bit of a hack, so open to other approaches to do this. I've seen people write a const anonymous funcs and wrap it all so you can one time call something like `<JenkinsLogo />` but thought that might be overkill here. And didn't bother extracting the height out to a style, either. It works, even if brute force ☺️

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4639-8uuyar0yieqxjmyuckgopnwyfi5t46tb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603466aa65f81755e9f74b43

**Comment by regicsolutions** [comment]
Looks great @adamdmharvey !

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4639-8uuyar0yieqxjmyuckgopnwyfi5t46tb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603466aa65f81755e9f74b43

File: plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx [hunk 2]
```diff
@@ -57,9 +55,10 @@ const BuildWithStepsView = () => {
   return (
     <div className={classes.root}>
       <Breadcrumbs aria-label="breadcrumb">
-        <Link to="../../..">Jobs</Link>
+        <Link to="../../..">Projects</Link>
         <Typography>Run</Typography>
       </Breadcrumbs>
+      <br />
```
Comment (freben) [inline plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx:61]: Probably just want to put the breadcrumbs in a wrapping div (or `<Box mb={1}>...` if you wanted some slight spacing) or so instead

File: plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx [hunk 3]
```diff
@@ -36,7 +35,6 @@ import ExternalLinkIcon from '@material-ui/icons/Launch';
 const useStyles = makeStyles(theme => ({
   root: {
     maxWidth: 720,
-    margin: theme.spacing(2),
```
Comment (freben) [inline plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx:39]: Makes me wonder, do people really like the maxWidth too? :) I know there's readability benefits with short lines but truncated/line-breaking things in tables aren't great either

File: plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx (hunk 1 above)
Comment (freben) [reply plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx:234]: This is fine!

Comment (freben) [reply plugins/jenkins/src/components/BuildsPage/lib/CITable/CITable.tsx:234]: as for alternatives ... i dunno, maybe a ListItem with a ListItemAvatar + Avatar? :) https://material-ui.com/components/lists/ If you like to experiment

File: plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx (hunk 3 above)
Comment (adamdmharvey) [reply plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx:39]: Yeah, great point.  If I take it out, it goes full width, and the table is sooooo small it's not really worth it.

Honestly, I think the table as a whole might be much better to be similar to an AboutCard and could be made more visual, but it just wouldn't be as simple as a table where we can add/remove columns.

So I think if it's OK for the purposes of this incremental update I won't touch it, but agree I think this entire page could do with some refactoring. Maybe check the GitHub Actions and CircleCI plugins and see if there could be some feature parity (and potentially reusable CI components) across them.

Comment (freben) [reply plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx:39]: Yep it was mostly a side note. Not a lot of other things use width-limiting.

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4639-8uuyar0yieqxjmyuckgopnwyfi5t46tb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603466aa65f81755e9f74b43

File: plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx (hunk 3 above)
Comment (adamdmharvey) [reply plugins/jenkins/src/components/BuildWithStepsPage/BuildWithStepsPage.tsx:39]: and oops, left my maxWidth testing in; reverted now

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4639-8uuyar0yieqxjmyuckgopnwyfi5t46tb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603466aa65f81755e9f74b43

**Review by freben — APPROVED** [review]
insert chef kissing fingers here

**Comment by regicsolutions** [comment]
@adamdmharvey i just tested the latest nightly build and getting a blank page.

image

**Comment by adamdmharvey** [comment]
@regicsolutions Just saw a user post something similar in Discord here:
https://discord.com/channels/687207715902193673/687235481154617364/815780294824165446

I wonder if this is something with webpack or a specific environmental combination. Any chance you can open a bug with some more context? like where you see this? on what page? Node version, that sort of thing?  May help track down the problem. I played around a few more times this eve but on my side can't reproduce what you're seeing. Looks like you're in the production mode, ya? i'd include any of those relevant details if you could in the bug report to see if it helps identify.  and worse case, if i've borked the plugin entirely for everyone, should be an easy roll back!

**Pull request merged**